eZ/Publish/API/Repository/SearchService.php

```diff
@@ -96,4 +97,13 @@ public function suggest($prefix, $fieldPaths = array(), $limit = 10, Criterion $
      * @return \eZ\Publish\API\Repository\Values\Content\Search\SearchResult
      */
     public function findLocations(LocationQuery $query, array $languageFilter = array(), $filterOnUserPermissions = true);
+
+    /**
+     * Get SortClause objects built from $location's sort options.
+     *
+     * @param \eZ\Publish\API\Repository\Values\Content\Location $location
+     *
+     * @return \eZ\Publish\API\Repository\Values\Content\Query\SortClause[]
+     */
+    public function getSortClauseFromLocation(Location $location);
```

Comment:
True, if we follow Symfony on this, so we should then introduce a new interface for this, make implementation implement both, and optionally clean up for 7.x then. right?